scripts/release/publish.js

```diff
@@ -40,7 +40,7 @@ try {
         `Attempting to release from branch "${releaseBranch}" using dist-tag "${distTag}".`
     );
 
-    execa.commandSync(`npm publish --tag ${distTag}`);
+    execa.commandSync(`npm publish --tag ${distTag} --registry=https://registry.npmjs.org`);
```

Comment:
why is not using this one by default?

Reply:
For whatever reason, CI is trying to publish to `registry.yarnpkg.com`. I checked to see if there was an environment variable or `.npmrc` file or something, but I'm not seeing it. So this is just me taking a shot in the dark and trying to fix it. 🤷